src/ophyd_async/epics/pvi/pvi.py

```diff
@@ -296,7 +301,9 @@ async def fill_pvi_entries(
 
 
 def create_children_from_annotations(
-    device: Device, included_optional_fields: Tuple[str, ...] = ()
+    device: Device,
+    included_optional_fields: Tuple[str, ...] = (),
+    device_vectors: Dict[str, int] = {},
 ):
     """For intializing blocks at __init__ of ``device``."""
```

Comment:
```suggestion
    device: Device,
    included_optional_fields: Tuple[str, ...] = (),
    device_vectors: Optional[Dict[str, int]] = None,
):
    """For intializing blocks at __init__ of ``device``."""
```

Avoiding mutable defaults.